Sources/MapboxNavigation/PuckType.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import MapboxMaps
+
+public enum PuckType {
+    
+    case puck2D(configuration: Puck2DConfiguration)
+    
+    case puck3D(configuration: Puck3DConfiguration)
+    
+    case `default`
```

Comment:
The default case should be the first one.

Reply:
Thanks for the suggestion, I'll improve the sequence and the documentation for this part.